### PR TITLE
[15.0][IMP] sale_elaboration: Add delivery date to the elaborations report

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @tecnativa TT49619

@CarlosRoca13 @sergio-teruel please review

### CODE REVIEW

File: sale_elaboration/models/sale_order.py [hunk 1]
```diff
@@ -58,6 +58,9 @@ class SaleOrderLine(models.Model):
         search="_search_is_prepared",
         help=("Dummy field to be able to find prepared lines"),
     )
+    scheduled_date = fields.Datetime(
```
Comment (sergio-teruel) [inline sale_elaboration/models/sale_order.py]: This field already exists here.. https://github.com/odoo/odoo/blob/15.0/addons/sale_stock/models/sale_order.py#L274

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

File: sale_elaboration/models/sale_order.py (hunk 1 above)
Comment (pilarvargas-tecnativa) [reply sale_elaboration/models/sale_order.py]: Right, I didn't check the dependency, thanks

**Review by sergio-teruel — APPROVED** [review]

File: sale_elaboration/i18n/es.po [hunk 2]
```diff
@@ -327,3 +327,14 @@ msgstr "Albarán"
...
+#~ msgid "Delivery date"
+#~ msgstr "Fecha de entrega"
+
+#~ msgid ""
+#~ "This is the delivery date promised to the customer. If set, the delivery "
+#~ "order will be scheduled based on this date rather than product lead times."
+#~ msgstr ""
+#~ "Es la fecha de entrega prometida al cliente. Si se establece, la orden de "
+#~ "entrega se programará en función de esta fecha y no de los plazos de "
+#~ "entrega del producto."
```
Comment (CarlosRoca13) [inline sale_elaboration/i18n/es.po]:
```suggestion

```

**Review by CarlosRoca13 — APPROVED** [review]
Minor comment

File: sale_elaboration/i18n/es.po (hunk 2 above)
Comment (pilarvargas-tecnativa) [reply sale_elaboration/i18n/es.po]: thanks

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Pull request merged**